CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+We welcome contributions via both the raising of issues and submitting pull requests. But before you
+do, please take a moment to consult the below.
+
+## Chatting
+
+If before any of the below you'd like to discuss an issue or some code, then come and chat with
+us on one of the following platforms:
+
+* [Users mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-users) - Most
+  active, best for more usage centric questions
+* [Developer mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-dev) - For
+  discussing work on the code
+* \#Equella channel in [Apereo's slack](https://apereo.slack.com/)
```

Comment:
https://apereo.slack.com/messages/C4KLR0C4U is the direct link to the Equella channel on Slack.

Reply:
Nice, I'll go and change that one. The above is just from the Apereo project site.